routes/game.rb

```diff
@@ -171,7 +171,7 @@ class Api
             description: r['description'],
             max_players: r['max_players'],
             settings: {
-              seed: Random.new_seed % 2**31,
+              seed: r['seed'] || Random.new_seed % 2**31,
```

Comment:
make it (r['seed'] || Random.new_seed) % 2 ** 31 so that we can sanitize the input a bit